src/components/Button/Button.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import "./Button.css"
-function Button(props){
+
+export default function Button(props){
```

Comment:
* Always separate imports and actual code with a space
* if you can, try to take advantage of all the features the programming language offers you, without sacrificing readability. In this case, `export default` on the same line looks cleaner. Anyone reading that will instantly now, that this is the parent component